content/doc/book/managing/plugins.adoc

```diff
@@ -225,6 +225,67 @@ The configuration(s) created by the disabled plugin behave as if the plugin
 were uninstalled, insofar that they result in warnings on boot but are
 otherwise ignored.
 
+==== Using the Jenkins CLI
+
+Making use of the <<cli#,Jenkins CLI>> it is also possible to enable and disable
```

Comment:
Would rephrase to *It is also possible to enable and disable plugins using the CLI* or similar.